src/llvm-coverage-instrumentation.md

```diff
@@ -452,30 +425,9 @@ significance.
 [simplify-cfg]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir_transform/simplify/enum.SimplifyCfg.html
 [rust-lang/rust#78544]: https://github.com/rust-lang/rust/issues/78544
 
-### `CoverageSpans`
-
-The `struct` [`CoverageSpans`][coverage-spans] builds and refines a final set of
-[`CoverageSpan`][coverage-span]s, each representing the largest contiguous `Span`
-of source within a single BCB. By definition--since each `Span` falls within a
-BCB, the `Span` is also non-branching; so if any code in that `Span` has executed,
-all code in the `Span` will have executed, the same number of times.
-
-[`CoverageSpans::generate_coverage_spans()`][generate-coverage-spans] constructs
-an initial set of `CoverageSpan`s from the `Span`s associated with each MIR
-`Statement` and `Terminator`.
-
-The final stage of `generate_coverage_spans()` is handled by
-[`to_refined_spans()`][to-refined-spans], which iterates through the `CoverageSpan`s,
-merges and de-duplicates them, and returns an optimal, minimal set of `CoverageSpan`s
-that can be used to assign coverage `Counter`s or `Expression`s, one-for-one.
-
-[coverage-spans]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir_transform/coverage/spans/struct.CoverageSpans.html
-[coverage-span]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir_transform/coverage/spans/struct.CoverageSpan.html
-[to-refined-spans]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir_transform/coverage/spans/struct.CoverageSpans.html#method.to_refined_spans
-
 ### `make_bcb_counters()`
 
-[`make_bcb_counters()`][make-bcb-counters] traverses the `CoverageGraph` and adds a
+[`make_bcb_counters`][] traverses the `CoverageGraph` and adds a
```

Comment:
Question: invalid link?

Reply:
This is shorthand for:

```markdown
[`make_bcb_counters`][`make_bcb_counters`]
```

Reply:
`[make_bcb_counters]` would also work

Reply:
The link already works as-is, so I'm not sure what I'm being asked to do here.